Cabal-syntax/src/Distribution/Pretty.hs

```diff
@@ -10,6 +10,8 @@ module Distribution.Pretty
   , showTokenStr
   , showFreeText
   , showFreeTextV3
+  , commaSpaceSep
```

Comment:
This should not pass formatting, or I am wrong in my understanding of fourmolu. Doesn't the exported items get sorted alphabetically?

Reply:
No. The export list also controls the ordering and grouping of the generated haddock.

Reply:
Ah my bad then. Stylish-Haskell does reorder the export list by default